projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -120,13 +120,9 @@ public function add_plugins_menu() {
 		// Calypso plugins screens link.
 		$plugins_slug = 'https://wordpress.com/plugins/' . $this->domain;
 
-		// `wpcomsh` restricts the plugins capabilities when the site doesn't have a
-		// supported plan so the Plugins menu it's not registered. We still want the
-		// menu to be available though, since it offers an opportunity to buy again
-		// the needed plan.
-		if ( ! $this->has_atomic_supported_plan() ) {
-			add_menu_page( 'Plugins', 'Plugins', 'manage_options', $plugins_slug, null, 'dashicons-admin-plugins', '65' );
-
+		// Link to the Marketplace on sites that can't manage plugins.
+		if ( ! wpcom_site_has_feature( \WPCOM_Features::MANAGE_PLUGINS ) ) {
```

Comment:
Since the Plugins menus is registered when the user has one of the `activate_plugins` capability](https://github.com/WordPress/WordPress/blob/fabc5db7fec6389fce09356a48980e6f04c622fe/wp-admin/menu.php#L302) and we disable that capability in `wpcomsh` when the site doesn't have the `INSTALL_PLUGINS` feature (see `wpcomsh_map_feature_cap`), should we check against `INSTALL_PLUGINS` instead?
```suggestion
		if ( ! wpcom_site_has_feature( \WPCOM_Features:: INSTALL_PLUGINS ) ) {
```

Reply:
We use [MANAGE_PLUGINS when redirecting to Calypso's plugin page](https://github.com/Automattic/wpcomsh/blob/master/feature-manager/feature-hooks.php#L128) which is why I used the same here. Should we separate out the `activate_plugins` cap and make it dependent on `MANAGE_PLUGINS`?

Reply:
> We use MANAGE_PLUGINS when redirecting to Calypso's plugin page in wpcomsh's `feature-manager/feature-hooks.php`

That seems partially wrong to me now that I'm looking at it closely. The redirect from `plugin-install.php` to `wordpress.com/plugins` should check `INSTALL_PLUGINS` because that's the feature that we check to disable the `install_plugins` capability which is the one that [registers the `plugin-install.php` menu](https://github.com/WordPress/WordPress/blob/fabc5db7fec6389fce09356a48980e6f04c622fe/wp-admin/menu.php#L308).

> Should we separate out the `activate_plugins` cap and make it dependent on `MANAGE_PLUGINS`?

That'd be great!

Reply:
Anyway, not a blocker for this PR since all those changes should happen in `wpcomsh`.